src/types.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+/* eslint-disable no-use-before-define */
```

Comment:
I'm fine with that. I think it's mainly for the use of `var` anyways, which we don't use.